lib/Ix/AccountState.pm

```diff
@@ -83,18 +83,15 @@ sub _save_states ($self) {
   my $rows = $self->_state_rows;
   my $pend = $self->_pending_states;
 
+  # XXX - Should only ever be one pending state. Croak if not true? Modify
+  #       code to expect only one? Other? -- alh, 2017-01-27
   for my $type (keys %$pend) {
     if (my $row = $rows->{$type}) {
       $row->update({ highestModSeq => $pend->{$type} });
     } else {
-      my $row = $self->schema->resultset('State')->create({
-        accountId => $self->accountId,
-        type      => $type,
-        highestModSeq => $pend->{$type},
-        lowestModSeq  => 0,
-      });
-
-      $rows->{$type} = $row;
+      die sprintf "State row for %s %s does not exist?!\n",
```

Comment:
Okay, we can't lock the collection without a state row, but I suspect that just means the call will fail if someone else creates the state row while we're in the middle of trying to